### DISCUSSION TITLE
Welcome to jtools Discussions!

### DISCUSSION THREAD

**Discussion by jacob-long** [jacob-long/jtools#125] [Announcements]
## 👋 Welcome!
  
I created discussions as a place for people to ask questions or leave comments that may not work as an issue. This should also make it easier for people besides me to chime in and help with those things.

**Comment by maozro** [comment]
Dear Jacob,
Thank you for a fantastic package! It is a true life saver. I could use help on the following issue:
I try to combine the "get_colors" option into a figure model. I have been trying to fit it in like this: 
"plot_BF<-plot_summs(MatchedCourtFullModel12,MatchedCourtFullModel13,MatchedCourtFullModel14,MatchedCourtFullModel15,MatchedCourtFullModel16,robust = TRUE, model.names = c("Legal", "Ideology", "Policy Agendas", "Intra Court", "Best Fit"),get_colors(colors=="Qual1", num.colors = 1, reverse = FALSE, gradient = FALSE))
plot_BF"
Which does not work. Any advice on how to fit the get_colors expression into the model? I basically need the figure in black and white.
Thanks in advance,
Maoz

**Comment by LucasKori** [comment]
Dear Jacob, 
without a doubt a great package. I wonder if there is a way to use the omit.coef = argument while also renaming the coefficients in the polt_sums() function... Currently they override each other. 
Thanks in Advance,
Lucas
